pages/security/human-security/passwords/fr.md

```diff
@@ -0,0 +1,64 @@
+@title = "Mots de passe"
+@toc = true
+@summary = "Mieux vivre en utilisant des mots de passe robustes"
+
+# Utiliser un gestionnaire de mots de passe
+
+Utiliser un gestionnaire de mots de passe représente un des changements importants à faire pour accroître votre sécurité  personnelle.
+
+Un gestionnaire de mots de passe permet l'utilisation de mots de passe qui sont à la fois forts et uniques.  En utilisant un gestionnaire de mots de passe on n'a qu'à se rappeler un seul mot de passe.  C'est le  gestionnaire qui contient tous les autres mots de passe.
+
+Il y a trois sortes de gestionnaire de mots de passe dont il faut être conscient.
+
+* **Application locale:** C'est un logiciel local qui entrepose tous les mots de passe sur votre ordinateur local, chiffrés avec le mots de passe maitre.  C'est le plus sûre parmis les trois options, mais il peut être difficile de synchroniser vos mots de passe entre de multiples appareils.
```

Comment:
I'm going to debate "sûre", which unmasks another error in my text:  the "le" should be "la" and it is "sûre" because it is one of the 3 options.  Which one? "La plus sûre des trois".  

So it would become "c'est la plus sûre parmis les trois options, ..."

How does that sound?

Reply:
Then I would go with "C'est l'option la plus sécuritaire des trois"

Reply:
fixed with "c'est l'option la plus sécuritaire des trois"